bahmni_atom_feed/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
These "atom feed" modules should not be needed. These modules don't care about atom feed AFAIK. They only receive REST API calls from the Java synchronisation bridge service.